src/lib/style/components/_bar-loader.scss

```
@@ -5,10 +5,18 @@
   height: 30px;
   width: 100%;
   .bar-background-color {
-    background-color: rgba(49, 95, 124, 0.05);
+    // background-color: rgba(49, 95, 124, 0.05);
+    background-color: #DFE2E6;
   }
   .bar-color {
-    background-color: #878787;
+    // background-color: #878787;
+    background-color: #F1D285;
```

```suggestion
    background-color:  map.get(SRC.$tertiary-color-palette, 400);
```

You may need to add a `!default` tertiary color palette to variables.scss. If you do, you can just copy the one from DemoStyle.scss